main/modes/games/2048/T48_leds.c

```diff
@@ -0,0 +1,229 @@
+/**
+ * @file T48_leds.c
+ * @author Jeremy Stintzcum (jeremy.stintzcum@gmail.com)
+ * @brief LED helpers for 2048
+ * @version 1.0.0
+ * @date 2024-08-16
+ *
+ * @copyright Copyright (c) 2024
+ *
+ */
+
+#include "T48_leds.h"
+
+void t48DimLEDs(t48_t* t48)
+{
+    for (uint8_t i = 0; i < CONFIG_NUM_LEDS; i++)
+    {
+        // Red
+        if (t48->leds[i].r < 6)
+        {
+            t48->leds[i].r = 0;
+        }
+        else if (t48->leds[i].r == 0)
```

Comment:
I don't think this will ever evaluate to true, since `< 6` would catch `== 0` above.